From cdc4bc6fa37e3d5173b3cad3d0f2e2d9816f8eb1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Sun, 15 Dec 2024 16:13:42 +0100
Subject: [PATCH] Drop rpm-vercmp requirement

It's a cmdline tool, part of rpmdevtools, not a python package.
---
diff --git a/requirements/base.txt b/requirements/base.txt
index 0061087e35..a42ff50f7d 100644
--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -40,7 +40,6 @@
 PyYAML
 requests<2.32.0 ; python_version < '3.10'
 requests>=2.32.5 ; python_version >= '3.10'
-rpm-vercmp; sys_platform == 'linux'
 setproctitle>=1.2.3
 timelib>=0.2.5; python_version < '3.11'
 timelib>=0.3.0; python_version >= '3.11'
-- 
2.46.0
